js/scripts.js

```diff
@@ -0,0 +1,8 @@
+
+var text = 'Papugi koszą trawę, unosząc się nad ziemią 15 centymetrów, na czerwonych dywanach'; 
+var animal = 'Zielone słonie';
+var animalUpperCased = animal.toUpperCase();
+var replacedAnimal = text.replace('Papugi', animalUpperCased);
+console.log(text.length);
+console.log(animalUpperCased.length);
+console.log(replacedAnimal.slice(0,45));
```

Comment:
Twój program zadziała tylko dla tego konkretnego zdania. Jeśli `text` ulegnie zmienie, to już nie wyświetlisz idealnie połowy tekstu. Spróbuj zrobić to tak żeby tekst dzielić na pół bez względu na to jak długi on będzie.